legion/legion/edi/server.py

```diff
@@ -229,12 +227,11 @@ def undeploy(model, version=None, grace_period=0, ignore_not_found=False):
             other_versions_exist = len(app.config['ENCLAVE'].get_models(model_service.id)) > 1
             LOGGER.info('Removing model\'s dashboard from Grafana (id={}, version={})'
                         .format(model_service.id, model_service.version))
-            if app.config['GRAFANA_CLIENT'].is_dashboard_exists(model_service.id, model_service.version) \
-                    and not other_versions_exist:
-                app.config['GRAFANA_CLIENT'].remove_dashboard_for_model(model_service.id,
-                                                                        model_service.version)
+
+            if not other_versions_exist:
+                app.config['GRAFANA_CLIENT'].remove_dashboard_for_model(model_service.id)
```

Comment:
I guess we have to keep `model_service.version` in remove function

Reply:
Why?

Reply:
Because if we remove this param this means that we constricts our code to specific realisation (that does not use model version).

Reply:
Done